README.md

```diff
@@ -66,7 +66,7 @@ docker tag docker.pkg.github.com/epiforecasts/covid-rt-estimates/covidrtestimate
 5. Update the estimates (see `docker logs covidrtestimates` for runtime information).
 
 ```bash
-sudo docker run -d --user rstudio --mount type=bind,source=$(pwd),target=/home/rstudio/covid-rt-estimates --name covidrtestimates covidrtestimates /bin/bash bin/update-estimates.sh
+sudo docker run -d --user rstudio --name covidrtestimates covidrtestimates /bin/bash bin/update-estimates.sh
```

Comment:
it's probably worth including a note to say this uses the code that ships in the docker image and if you want to use your own version mount it in yourself

Reply:
(just to state the obvious but assume a non-technical person may be trying to follow the readme)

Reply:
Done